packages/wallet-sdk/src/connection/WalletLinkConnection.ts

```diff
@@ -66,7 +57,25 @@ export class WalletLinkConnection {
   private nextReqId = IntNumber(1);
   private connectedSubject = new BehaviorSubject(false);
   private linkedSubject = new BehaviorSubject(false);
-  private sessionConfigSubject = new ReplaySubject<SessionConfig>(1);
+
```

Comment:
these listeners below will be replaced later, once rxjs is removed from this class.